cli/src/commands/create/quota.rs

```diff
@@ -0,0 +1,61 @@
+use anyhow::{bail, Context, Result};
+use log::info;
+use reinfer_client::{
+    resources::{
+        quota::{CreateQuota, TenantQuotaKind},
+        tenant_id::{ReinferTenantId, TenantId, UiPathTenantId},
+    },
+    Client,
+};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct CreateQuotaArgs {
+    #[structopt(long = "reinfer-tenant-id")]
+    /// Reinfer tenant ID for which to set the quota
+    reinfer_tenant_id: Option<ReinferTenantId>,
+
+    #[structopt(long = "uipath-tenant-id")]
+    /// UiPath tenant ID for which to set the quota
+    uipath_tenant_id: Option<UiPathTenantId>,
+
+    #[structopt(long = "quota-kind")]
+    /// Kind of quota to set
+    tenant_quota_kind: TenantQuotaKind,
+
+    #[structopt(long = "limit")]
+    /// New value of the quota to set
+    hard_limit: u32,
+}
+
+pub fn create(client: &Client, args: &CreateQuotaArgs) -> Result<()> {
+    let CreateQuotaArgs {
+        reinfer_tenant_id,
+        uipath_tenant_id,
+        tenant_quota_kind,
+        hard_limit,
+    } = args;
+
+    let tenant_id: TenantId = match (reinfer_tenant_id, uipath_tenant_id) {
+        (Some(tenant_id), None) => TenantId::Reinfer(tenant_id.to_owned()),
+        (None, Some(tenant_id)) => TenantId::UiPath(tenant_id.to_owned()),
```

Comment:
Instead of this you should be able to do this with an [argument group](https://github.com/TeXitoi/structopt/issues/104#issuecomment-907318718).